bots/migrations/0122_tag_featured_tag_hide_tag_landing_page_and_more.py

```
@@ -0,0 +1,47 @@
+                help_text="Optional landing page URL for this tag.",
+                max_length=2048,
+            ),
+        ),
+        migrations.AlterField(
+            model_name="tag",
+            name="featured_priority",
+            field=models.IntegerField(
+                default=1, help_text="Higher priority tags are shown first."
+            ),
+        ),
+    ]
```

Ruff (0.15.15)

[warning] Mutable default value for class attribute

(RUF012)